src/fastoad/model_base/flight_point.py

```diff
@@ -198,7 +271,45 @@ def get_units(cls) -> dict:
 
         A dimensionless physical quantity will have "-" as unit.
         """
-        return cls._units
+        return {
+            name: field_descriptor.unit
+            for name, field_descriptor in cls._get_field_descriptors().items()
+        }
+
+    @classmethod
+    def get_unit(cls, field_name) -> Optional[str]:
+        """
+        Returns unit for asked field.
+
+        A dimensionless physical quantity will have "-" as unit.
+        """
+        return cls._get_field_descriptors().get(field_name, _FieldDescriptor()).unit
```

Comment:
If I understand correctly, this syntax makes it so that if the field_name is not in the __field_descriptors we get a None ? Wasn't there a more readable way ?

Reply:
Also why not _FieldDescriptor(None, None, None) as is done below ?